pkg/rtc/participant.go

```diff
@@ -245,7 +245,8 @@ type ParticipantImpl struct {
 	onDataMessage        func(types.LocalParticipant, []byte)
 	onMetrics            func(types.Participant, *livekit.DataPacket)
 
-	migrateState atomic.Value // types.MigrateState
+	migrateState                   atomic.Value // types.MigrateState
+	migratedTracksPublishedPromise *utils.Promise[bool]
```

Comment:
it doesn't look like we're using the value for anything. would a fuse work here?

Reply:
Yeah, good point, let me re-do this.